src/app/constants/coordinates.ts

```diff
@@ -0,0 +1,4 @@
+export class Coordinates {
+    lat: number;
+    lng: number;
+}
```

Comment:
Use an interface when you want to define the shape of an object without providing any implementation